content/blog/2022-04-05-newsletter.de.md

```diff
@@ -0,0 +1,188 @@
+---
```

Comment:
The "frontmatter" has to be left untouched. Frontmatter is everything between those `---`.